site/docs/components/button/accessibility.mdx

```diff
@@ -1,35 +1,45 @@
 ---
-title: Button
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
 layout: DetailComponent
 sidebar:
   exclude: true
 data:
   $ref: ./#/data
 ---
 
-### Best practises
+### Best practices
 
-#### When a Button has both an icon and a label
+- When a Button has both an icon and text, pass `aria-hidden=”true”` to the icon so that its text is not announced by screen readers.
+- When a Button has an icon only and no text, pass an `aria-label` to Button that describes its purpose.
 
-- Pass `aria-hidden=”true”` to the icon so that its label is not announced by screen readers.
+#### Focusable disabled button
 
-#### When a Button has an icon only and no label
+- By default, a disabled button is not focusable. However, in certain scenarios it may be preferable for a disabled button to receive focus. To determine if this is necessary, the following rule of thumb can be followed: "If a disabled element can be enabled (either by the user or by the system) it should be made focusable.” For further information, view the W3C guidance on the [`aria-disabled` attribute](https://www.w3.org/TR/wai-aria/#aria-disabled).
 
-- Pass an `aria-label` to Button that describes its purpose.
+#### Screen reader behavior:
+
+On focus, the screen reader should:
+
+- Identify the button
+- Read out the text,
+- Express the state
+  - In certain situations, the screen reader may need to indicate if the button has a popup for lists or menus.
```

Comment:
```suggestion
- In certain situations, the screen reader may need to indicate if the button has a popup for lists or menus.
```

Reply:
the tab was intended as this is a sub-point for express the state